frontend/test/__support__/e2e/cypress.js

```diff
@@ -321,3 +321,11 @@ export function mockSessionProperty(propertyOrObject, value) {
     });
   });
 }
+
+export function mockSessionProperty(propertyOrObject, value) {
+  mockProperty(propertyOrObject, value, "/api/session/properties");
+}
```

Comment:
@daltojohnso FYI I needed a similar function (just a different URL), so I extracted the logic into base function `mockProperty`. It is now being used for `mockSessionProperty()` and `mockCurrentUserProperty()`. The existing functionality shouldn't change. Just wanted to let you know since you created this function originally.
Thank you for that. It made my life easier on numerous occasions already.